web-api/terraform/template/main-west.tf

```diff
@@ -129,6 +143,8 @@ data "aws_s3_bucket_object" "puppeteer_green_west_object" {
 data "aws_elasticsearch_domain" "green_west_elasticsearch_domain" {
   depends_on = [
     aws_elasticsearch_domain.efcms-search,
+    module.elasticsearch_alpha,
```

Comment:
I couldn't map what issue these updates are addressing, is it one of the ones in already in the PR description?

Reply:
It's connected to https://github.com/flexion/ef-cms/pull/6945

Reply:
@julialeague Right now we have a bunch of versions 1,2,3,4 for our tables + elasticsearch which is getting confusing.  We talked about just switching to 2 tables, alpha and beta, and when a migration is needed, we'd switch from alpha -> beta or beta -> alpha instead of making all these new table version.

The plan is to remove these versioned tables in the new court PR or so.